ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -535,4 +537,30 @@ - (RTCMediaStreamTrack *)trackForId:(nonnull NSString *)trackId pcId:(nonnull NS
     return peerConnection.remoteTracks[trackId];
 }
 
+- (void)ensureAudioSessionWithRecording {
+  RTCAudioSession* session = [RTCAudioSession sharedInstance];
+  // we also need to set default WebRTC audio configuration, since it may be activated after
+  // this method is called
+  RTCAudioSessionConfiguration* config = [RTCAudioSessionConfiguration webRTCConfiguration];
+  // require audio session to be either PlayAndRecord or MultiRoute
+  if (session.category != AVAudioSessionCategoryPlayAndRecord &&
+      session.category != AVAudioSessionCategoryMultiRoute) {
```

Comment:
I believe that this one isn't necessary. Multi-route category is for mixer apps and they don't really seem to fall in the spectrum of our usecases.